www/hugo/Portfile

```diff
@@ -46,6 +47,13 @@ destroot {
     xinstall -m 755 ${workpath}/bin/${name} ${destroot}${prefix}/bin/${name}
 }
 
+post-destroot {
+    # generate man pages then install them
+    system -W ${workpath} "${workpath}/bin/${name} gen man"
+    xinstall -d -m 0755 ${destroot}${prefix}/share/man/man1
+    xinstall -m 0644 {*}[glob ${workpath}/man/*.1] ${destroot}${prefix}/share/man/man1/
```

Comment:
My Tcl knowledge is a bit weak, so someone else will have to say if the `{*}` bit is correct.

Reply:
Ah, so it is indeed correct. Ignore my comment.